database/DDL_statements.sql

```diff
@@ -53,22 +53,22 @@ CREATE TABLE ScheduledDosages(
 
 CREATE TABLE Videos(
        id		INT		NOT NULL PRIMARY KEY AUTO_INCREMENT,
-       topic		VARCHAR(255) 	NOT NULL,
-       title		VARCHAR(255) 	NOT NULL,
+       topic		VARCHAR(255) 	NOT NULL  UNIQUE,
```

Comment:
Also I just remembered, topic cannot have the `UNIQUE` constraint, since some videos have the same topic.